### PR TITLE
Update docs to include Policy Cloning API

### PR DESCRIPTION
**Pull request by justinshih-okta**
## Description:
- Added api documentation regarding a new OIE policy endpoint: Policy Cloning
- Updated the policy postman collection to include policy cloning
- **Related to Monolith release:** 2022.09.0
image

### Resolves:

* [OKTA-522604](https://oktainc.atlassian.net/browse/OKTA-522604)

### CODE REVIEW

**Review by iancorcoran-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -198,6 +198,31 @@ curl -v -X POST \
...
+### Clone a Policy
+
+<ApiLifecycle access="ie" />
+
+> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+> **Note:** Within the Identity Engine, this feature is only supported for [Authentication Policies](#authentication-policy).
+
+<ApiOperation method="post" url="/api/v1/policies/${policyId}/clone" />
+
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:210]: Be good to be consistent and include the Request parameters section and call out that the Policy ID is required:

#### Request parameters

The Policy ID described in the [Policy object](/docs/reference/api/policy/#policy-object) is required.

**Review by susanharper-okta — COMMENTED** [review]
two comments

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 2]
```diff
@@ -198,6 +198,35 @@ curl -v -X POST \
...
+> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+> **Note:** Within the Identity Engine, this feature is only supported for [authentication Policies](#authentication-policy).
+
+<ApiOperation method="post" url="/api/v1/policies/${policyId}/clone" />
+
+##### Request parameters
+
+The policy id described in the [Policy object](#policy-object) is required.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:213]:
```suggestion
The policy ID described in the [Policy object](#policy-object) is required.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 3]
```diff
@@ -198,6 +198,35 @@ curl -v -X POST \
 HTTP 204:
 [Policy object](#policy-object)
 
+### Clone a Policy
+
+<ApiLifecycle access="ie" />
+
+> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+> **Note:** Within the Identity Engine, this feature is only supported for [authentication Policies](#authentication-policy).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:207]:
```suggestion
> **Note:** Within the Identity Engine, this feature is only supported for [authentication policies](#authentication-policy).
```

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**